tests/func/test_data_cloud.py

```diff
@@ -68,6 +68,8 @@ def _check_status(status, **kwargs):
             expected = kwargs.get(key, set())
             assert expected == {obj.hash_info for obj in getattr(status, key)}
 
+    odb = dvc.cloud.get_remote_odb("upstream")
+    assert not odb.exists(foo_hash)
```

Comment:
Hm, very interesting that this gets triggered, even though we are using unique urls :thinking:

Reply:
Yeah I was initially thinking it that we were maybe hitting UUID collisions w/the URL generation, but it seems like it's probably something gdrive internals specific. I'll keep digging into it

Reply:
@pmrowla Maybe let's just temporarily xfail or skip this test for gdrive and create a ticket. We'll be migrating gdrive to fsspec soon, and might take a look then.